src/util/getUrl.ts

```diff
@@ -11,7 +11,7 @@ export const getGitHubConfigurationUrl = (urlParams: UrlParams): string => {
 };
 
 export const getJiraMarketplaceUrl = (jiraHost: string): string =>
```

Comment:
nit: Would it make sense to use "jiraBaseUrl" as the name instead?

Reply:
Hmm I think we should leave as is because it directs users to our app in the Jira marketplace.

Reply:
Yeah, sorry, I'm not after changing the logic :) Just the name itself. Host means "blahblah.atlassian.net" ; baseurl means "https://blahblah.atlassian.net" . In this code I think we wait for the baseUrl rather than the host...